doc/source/tune/user-guide.rst

```diff
@@ -527,6 +527,15 @@ By default, Tune will run hyperparameter evaluations on multiple processes. Howe
 
 Local mode with multiple configuration evaluations will interleave computation, so it is most naturally used when running a single configuration evaluation.
 
+Note that ``local_mode`` has some known issues, so please read carefully before using. You should not use ``local_mode`` if:
+1. If your application depends on setting environment variables per process
+2. If your application has recursive actor calls
+3. If your remote actor/task sets any sort of process-level global variables
+4. If you use are using async actors
+
+Also note, if you are using GPUs, you must set the ``CUDA_VISIBLE_DEVICES`` environment
+variable to a comma separated list of your GPU Device IDs.
```

Comment:
Instead of replicating this, can you make a reference to the `local_mode` section of `starting-ray.rst`? You'll want to use https://www.sphinx-doc.org/en/1.7/markup/inline.html#cross-referencing-arbitrary-locations again